server/__init__.py

```diff
@@ -219,18 +219,18 @@ def register_routes_common(app):
 def create_app():
   app = Flask(__name__, static_folder='dist', static_url_path='')
 
-  if os.environ.get('FLASK_ENV') in ['production', 'staging', 'autopush']:
-    createMiddleWare(app, StackdriverExporter())
-    import googlecloudprofiler
-
-    # Profiler initialization. It starts a daemon thread which continuously
-    # collects and uploads profiles. Best done as early as possible.
-    try:
-      # service and service_version can be automatically inferred when
-      # running on GCP.
-      googlecloudprofiler.start(verbose=3)
-    except (ValueError, NotImplementedError) as exc:
-      logging.error(exc)
+  # if os.environ.get('FLASK_ENV') in ['production', 'staging', 'autopush']:
```

Comment:
should we just delete this if cloudprofiler isn't being used?

Reply:
Removed